cogs/utils/generic.py

```diff
@@ -60,9 +65,29 @@ async def get_channels(self, ctx):
         await ctx.channel.send(f"```{output}```")
 
 
-    # Do stuff when a message is deleted
-    @commands.Cog.listener()
-    async def on_message_delete(self, message):
-        pass
```

Comment:
This placeholder is for eventual logging functions related to preserving deleted messages. Though it doesn't do anything at the moment, it doesn't hurt anything to leave it in, so no reason to remove it at this point.